src/Symfony/Component/Validator/Mapping/Loader/PropertyInfoLoader.php

```diff
@@ -60,6 +60,10 @@ public function loadClassMetadata(ClassMetadata $metadata)
                 continue;
             }
 
+            if (false === property_exists($className, $property)) {
```

Comment:
property_exists() is what is used in [PropertyMetadata](https://github.com/symfony/symfony/blob/c5a29278e212659cdc0f17f148fc4a252c669983/src/Symfony/Component/Validator/Mapping/PropertyMetadata.php#L39), which triggered the exception. This matches that.